mobile_back_tflite/cpp/backend_tflite/tflite_settings_apple.h

```diff
@@ -31,36 +31,36 @@ common_setting {
 
 benchmark_setting {
   benchmark_id: "image_classification"
-  accelerator: "coreml"
-  accelerator_desc: "CoreML"
-  configuration: "TFLite"
+  accelerator: "ane"
+  accelerator_desc: "ANE"
+  framework: "TFLite CoreML"
```

Comment:
You should also update the `coreml` check in tflite backend to match new settings, or revert it.

Reply:
Done